### PR TITLE
[zio-hackathon-2022] Add current_time function to MysqlSqlModule

### PR DESCRIPTION
**Pull request by amrkamel**
Resolves https://github.com/zio/zio-sql/issues/653

### CODE REVIEW

File: mysql/src/main/scala/zio/sql/mysql/MysqlSqlModule.scala [hunk 1]
```diff
@@ -27,12 +26,13 @@ trait MysqlSqlModule extends Sql { self =>
...
-    val Log10     = FunctionDef[Double, Double](FunctionName("log10"))
-    val Pi        = Expr.FunctionCall0[Double](FunctionDef[Any, Double](FunctionName("pi")))
-    val BitLength = FunctionDef[String, Int](FunctionName("bit_length"))
+    val Crc32       = FunctionDef[String, Long](FunctionName("crc32"))
+    val Degrees     = FunctionDef[Double, Double](FunctionName("degrees"))
+    val Log2        = FunctionDef[Double, Double](FunctionName("log2"))
+    val Log10       = FunctionDef[Double, Double](FunctionName("log10"))
+    val Pi          = Expr.FunctionCall0[Double](FunctionDef[Any, Double](FunctionName("pi")))
+    val BitLength   = FunctionDef[String, Int](FunctionName("bit_length"))
+    val CurrentTime = Expr.ParenlessFunctionCall0[OffsetTime](FunctionName("current_time"))
```
Comment (baldram) [inline mysql/src/main/scala/zio/sql/mysql/MysqlSqlModule.scala:35]: How about considering the alphabetical order here? I see that the `BitLength` broke it a bit. Now one more. 

It's useful when considering the code completion feature and what suggestions the library user sees.

Comment (amrkamel) [reply mysql/src/main/scala/zio/sql/mysql/MysqlSqlModule.scala]: Yup makes perfect sense :+1: . fixed

**Comment by jczuchnowski** [comment]
@amrkamel @ri-amralloush you need to accept from the second account :)

**Comment by amrkamel** [comment]
closed because of the double users issue, I can't access the second user atm. will release a new PR

**Pull request closed without merging**